packages/flutter/test/semantics/semantics_update_test.dart

```diff
@@ -0,0 +1,188 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:typed_data';
+import 'dart:ui' as ui;
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test('Semantics update', () async {
```

Comment:
Any reasons this cannot be a regular `testWidgets` test? You should be able to just instantiate the custom binding in the main before all `testWidgets` methods.